main.py

```diff
@@ -725,24 +802,46 @@ def _send_flow_mods(
         flows,
         reraise_conn=True,
         send_barrier=ENABLE_BARRIER_REQUEST,
+        batch_size=0,
+        batch_interval=0,
     ):
         """Send FlowMod (and BarrierRequest) given a list of flow_dicts to switches."""
+        flows_zipped = list((fmod, flow) for fmod, flow in zip(flow_mods, flows))
+        batch_size = (
+            min(batch_size, len(flows_zipped))
+            if batch_size > 0
+            else max(1, len(flows_zipped))
+        )
+        batch_interval = max(0, batch_interval)
         for switch in switches:
```

Comment:
Nothing is stopping multiple requests from sending in flow_mods to the same switch, which could potentially overload the switch. Perhaps instead of performing batching here, we could use a per switch queue for the flow_mods received to rate limit.
Additionally, batch size doesn't make much sense as a per request parameter, but makes sense as a per switch parameter.

Reply:
Yes, @Ktmi that's a good idea. We might as well go ahead and start queueing per switch on flow_manager. Initially this was just adding the same client per request behavior. I'll augment this shortly. 

It's worth also nothing that we also have in our backlog to have per message type to support rate limiting on core queue but hasn't been priorized https://github.com/kytos-ng/kytos/issues/245

Reply:
@Ktmi check out this thread https://github.com/kytos-ng/kytos/issues/245#issuecomment-1711685003, I'll need your help with this one if we'll go for the ideal solution, appreciated your review, help, and suggestions. In the meantime, I'll keep making progress on `telemetry_int`. Otherwise, I can still handle it in the future. If you confirm that you can help out with that one on version `2023.2`, of course following the current planned priorities, we can close this PR here. Let me know.

Reply:
Closing this PR in favor of https://github.com/kytos-ng/kytos/pull/412. David is helping out with it. Thanks, David.